storage/perfschema/table_variables_by_thread.cc

```diff
@@ -102,7 +102,7 @@ PFS_engine_table *table_variables_by_thread::create(PFS_engine_table_share *) {
 ha_rows table_variables_by_thread::get_row_count(void) {
   mysql_mutex_lock(&LOCK_plugin_delete);
 #ifndef NDEBUG
-  extern mysql_mutex_t LOCK_plugin;
+  extern mysql_mutex_t LOCK_plugin [[maybe_unused]];
```

Comment:
PS code does not have [[maybe_unused]] attribute. Why do we need it for PXC?
It should be in WITH_WSREP block.